Source/Core/Common/StringUtil.h

```diff
@@ -14,8 +14,13 @@
 #include <type_traits>
 #include <vector>
 
+#include "Common/CommonFuncs.h"
 #include "Common/CommonTypes.h"
 
+#ifndef _WIN32
+#include <strings.h>
+#endif
```

Comment:
It's *not* included on Windows (since it's #if<ins><b>n</b></ins>def).  Also note that it's string<ins><b>s</b></ins>.h, not string.h; [strings.h defines non-standard functions such as strcasecmp](https://stackoverflow.com/a/4291328).

It works on Windows because `CommonFuncs.h` provides a `#define strcasecmp _stricmp`.